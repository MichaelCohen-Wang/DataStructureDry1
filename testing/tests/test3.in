add_ship 155356 26
add_pirate 451855 155356 25
add_pirate 211260 155356 24
update_pirate_treasure 211260 30
update_pirate_treasure 451855 28
get_treasure 211260
remove_pirate 211260
add_pirate 221689 155356 25