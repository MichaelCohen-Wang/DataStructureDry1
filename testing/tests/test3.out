add_ship: SUCCESS
add_pirate: SUCCESS
add_pirate: SUCCESS
update_pirate_treasure: SUCCESS
update_pirate_treasure: SUCCESS
get_treasure: SUCCESS, 54
remove_pirate: SUCCESS
add_pirate: SUCCESS
